drv/transceivers-server/src/main.rs

```diff
@@ -70,13 +72,23 @@ ringbuf!(Trace, 16, Trace::None);
 
 ////////////////////////////////////////////////////////////////////////////////
 
+// 0-31 for port LEDs, 32 is System LED
```

Comment:
This comment is confusing here, because it implies there are 33 items in the array below (but `NUM_PORTS` is only 32, and the system LED is actually stored separately in `system_led_state`).

Reply:
Yeah that is me missing cleanup from a prior implementation. Sorry for missing that!